AfrikaBurn/Model/AfrikaBurnElement.swift

```diff
@@ -13,21 +13,51 @@ extension AfrikaBurnElement.ElementType {
     
     init?(name: String) {
         switch name.lowercased() {
-        case "mutant vehicles":
+        case "mutant vehicles", "mutantvehicle":
             self = .mutantVehicle
-        case "performance registration":
+        case "performance", "performance registration":
             self = .performance
-        case "theme camp form 3 - wtf guide":
+        case "camp", "theme camp form 3 - wtf guide":
             self = .camp
-        case "artwork registration":
+        case "artwork", "artwork registration":
```

Comment:
I was getting crashes on launch saying ```received an unknown element type "artwork"```
Adding this stopped the crashes. I'm thinking maybe they renamed them on the server?